apps/src/templates/studioHomepages/TwoColumnActionBlock.jsx

```diff
@@ -239,8 +242,31 @@ export class SpecialAnnouncementActionBlock extends Component {
       </div>
     );
 
-    return (
+    return !!this.props.hasIncompleteApplication ? (
+      <TwoColumnActionBlock
+        id="continue-teacher-application-announcement"
+        imageUrl={pegasus(
+          '/shared/images/fill-540x289/teacher-announcement/professional-learning-2019-3.jpg'
+        )}
+        subHeading={
+          'Finish your application to the Professional Learning Program'
+        }
+        subHeadingSmallFont={true}
+        description={
+          'We noticed you started your application to the Code.org Professional Learning Program.\
+          Finish your application while seats last!'
+        }
+        buttons={[
+          {
+            id: 'continue-teacher-application-button',
+            url: '/pd/application/teacher',
+            text: 'Finish application'
+          }
+        ]}
+      />
+    ) : (
       <TwoColumnActionBlock
+        id="teacher-nomination-announcement"
```

Comment:
Oh, the two IDs could be a little more consistent, say: `teacher-nomination-announcement` and `teacher-nomination-continue-button`.

Reply:
ah year, will fix